### PR TITLE
NN-4603 YourCompletedReport update

### PR DESCRIPTION
**Pull request by HamHazJustice**

### CODE REVIEW

**Comment by HamHazJustice** [comment]
**_Note the screen designs have not been updated but the Acceptance Criteria are inline with AllCompletedReports updates in order to be more uniform in look&feel_**

File: integration_tests/integration/yourCompletedReports.cy.ts [hunk 1]
```diff
@@ -25,12 +26,15 @@ context('Your Completed Reports', () => {
   it('pagination should work', () => {
     const manyReportedAdjudications: ReportedAdjudication[] = generateRange(1, 300, _ => {
       return {
+        gender: PrisonerGender.MALE,
         adjudicationNumber: _,
         prisonerNumber: 'A1234AA',
         bookingId: 1,
+        dateTimeOfDiscovery: '2021-11-15T11:30:00',
```
Comment (natclamp-moj) [inline integration_tests/integration/yourCompletedReports.cy.ts:33]: I don't think this needs to be here - you've already added it into the incidentDetails object below

File: integration_tests/integration/yourCompletedReports.cy.ts [hunk 2]
```diff
@@ -137,17 +143,30 @@ context('Your Completed Reports', () => {
...
   })
 
   it('filtering and pagination should work together', () => {
     const manyReportedAdjudications: ReportedAdjudication[] = generateRange(1, 300, _ => {
       return {
+        gender: PrisonerGender.MALE,
         adjudicationNumber: _,
         prisonerNumber: 'A1234AA',
         bookingId: 1,
+        dateTimeOfDiscovery: '2021-11-15T11:30:00',
```
Comment (natclamp-moj) [inline integration_tests/integration/yourCompletedReports.cy.ts:165]: Doesn't need to be here

File: server/views/pages/yourCompletedReports.njk [hunk 3]
```diff
@@ -46,18 +45,17 @@
...
       head: [
-        { html: 'Prisoner&#8217;s name' },
-        { text: "Prison number" },
-        { text: "Date and time of incident",
+        { text: "Discovery date and time",
           attributes: {
           "aria-sort": "descending"
         }
         },
+        { html: 'Name and prison number' },
```
Comment (natclamp-moj) [inline server/views/pages/yourCompletedReports.njk:53]: I think this can just be 'text' rather than html
`{ text: 'Name and prison number' },`

**Review by natclamp-moj — APPROVED** [review]
LGTM

**Pull request merged**